webpack.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
     output: {
         path: path.resolve(__dirname, 'deploy/scripts'),
         filename: 'ancientbeast.js'
-    }
+    },
+    devtool: 'cheap-source-map'
```

Comment:
According to webpack documentation, this option is not ideal for development or production, what is the reason for choosing it over a potentially better suited option?

Reply:
It was the only one that actually worked for me. Sourcemaps are kinda buggy with chrome. I will look into this for the future. There needs to be another webpack file for production tough. However, I'm a big believer in MVP (minimum viable product) and LEAN. The sooner this branch get merged, the sooner I can add those extra features. I want to get this working first.

Reply:
That's fine, this was mostly a curiosity comment because of how the docs were worded. Completely agree on getting things merged as quick as possible, moving things to mostly ES6 faced a similar problem, so completely understand.